assets/src/scss/layout/_navbar.scss

```diff
@@ -261,24 +324,24 @@ a.nav-link {
     transform: scaleX(0);
     transition: transform 0.25s;
     z-index: -1;
+
+    @include large-and-up {
+      border-bottom-width: 4px;
+    }
   }
 }
 
 .active a.nav-link {
   --nav-link-active-- {
```

Comment:
Removed all this code since the border bottom is assigned to the `:after` pseudo element.